frontend/cli/cmd_interactive.go

```diff
@@ -62,3 +74,72 @@ func (i *interactiveCmd) Run(ctx context.Context, k *kong.Kong, projectConfig pr
 func errorf(format string, args ...any) {
 	fmt.Printf("\033[31m%s\033[0m\n", fmt.Sprintf(format, args...))
 }
+
+func (f *FTLCompletion) Do(line []rune, pos int) ([][]rune, int) {
+	parser := f.app
+	if parser == nil {
+		return nil, 0
+	}
+	all := []string{}
+	completed := []string{}
+	last := ""
+	lastCompleted := ""
+	lastSpace := false
+	// We don't care about anything past pos
+	// this completer can't handle completing in the middle of things
```

Comment:
Yeah I've noticed that before, it sucks a bit :( - still, better than no completion